src/pages/RegisterEmailPage.tsx

```diff
@@ -0,0 +1,42 @@
+import "../styles/RegisterEmailPage.scss"
+
+import {ChangeEvent, useState} from 'react';
+
+import AppBackButton from '../components/atoms/AppBackButton';
+import AppContinueButton from '../components/atoms/AppContinueButton';
+import EmailInput from "../components/atoms/EmailInput";
+
+const RegisterEmailPage = () => {
+  const [emailValidity, setValidity] = useState(false)
+
+  const validEmail = (event: ChangeEvent<HTMLInputElement> ) => {
+    const email = event.target;
+
+    if (email.value == "") {
+      return false
+    }
+    else {
+      return email.validity.valid
+    }
```

Comment:
Would it make more sense to check whether email.value is different to empty, undefined, null, false, etc.?